opencog/guile/SchemeSmobTV.cc

```diff
@@ -460,5 +460,32 @@ SCM SchemeSmob::ss_tv_get_value (SCM s)
 	return SCM_EOL;
 }
 
+/**
+ * Return the truth value mean
+ */
+SCM SchemeSmob::ss_tv_get_mean(SCM s)
+{
+	TruthValue *tv = verify_tv(s, "cog-tv-mean");
+	return scm_from_double(tv->getMean());
+}
```

Comment:
well, I don't think its limited to guile-c++, similar numbers apply to cython/python. 

Hmm, as I write this, I realize some of the numbers I quoted are wrong.

I'm thinking that I might have mis-quoted the numbers, -- there's a big difference between entering gule for the first time, and entering it again a second time -- there's a fast track for the second time, since everything is already set up. Likewise for the reverse direction.   You'd have to measure carefully and ponder.